snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -25,24 +26,45 @@
 class Multipass(Provider):
     """A multipass provider for snapcraft to execute its lifecycle."""
 
+    @classmethod
```

Comment:
This is not static. ABC didn't catch that, eh?

Reply:
issue taken care of